spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Smoke test for demo app', function() {
+    
+    browser.waitForAngularEnabled(false);
+    browser.manage().window().maximize();
+    browser.get('http://automationpractice.com/index.php?controller=authentication&back=my-account');
+    
+
+it('Initial build testing', function() {
+    
+    // Enter email in "Email address" field, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("email")).sendKeys("gigedov@getnada.com");
+
+    // Enter password in "Password" field, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("passwd")).sendKeys("testing1");
+
+    // Click on "Sign in" button, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("SubmitLogin")).click();
+    browser.sleep(3000);
```

Comment:
browser.sleep is not good practice,  can you try modify your code to not use sleep ?
Instead of sleep you should wait for certain element to be displayed or clickable.
Some ideas:
https://stackoverflow.com/questions/47135945/how-to-wait-for-any-element-to-appear-in-protractor?rq=1
https://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.elementToBeClickable